spec/views/users/edit.html.erb_spec.rb

```diff
@@ -14,16 +14,26 @@
     expect(rendered).to have_css("h1.govuk-heading-l", text: /Edit user/)
   end
 
-  it "contains name" do
-    expect(rendered).to have_text(user.name)
-  end
+  describe "summary list" do
+    let(:summary_list) do
+      Capybara.string(rendered).find(".govuk-summary-list")
+    end
```

Comment:
I love how you have scoped the component here to test the various attributes being played back on the page